decidim-bulletin-board-client-js/src/client/graphql-client.js

```diff
@@ -92,7 +91,12 @@ export class GraphQLClient {
           electionId,
         },
       })
-      .pipe(map(({ data }) => data.electionLogEntryAdded.logEntry));
+      .map(
+        ({ data }) =>
+          data &&
+          data.electionLogEntryAdded &&
+          data.electionLogEntryAdded.logEntry
```

Comment:
Actually... is this value that was `{}` the first time you subscribe 😅 . It would be nice having types here so I don't need to be that extra careful haha.